po/LINGUAS

```diff
@@ -1,25 +1,11 @@
 ca
-cs
```

Comment:
The commit description is a bit misleading. These languages (currently) *do* get used in Cockpit, if the browser requested language matches.  You just can't select them in the dialog, but that's just a corner case anyway (we expect that in most cases the desktop/browser language is the appropriate default). Perhaps this commit should go after the "50% rule"?

Also, why isn't this just a result of running the bot once, when it would automatically remove these languages?

Reply:
Yes, it removes all languages, that do not appear in the manifest (and therefore are not displayed in Cockpit). Bot is not able to add file into manifest that was not newly pulled (I could write that but it is pointless, because it should never happen). A language needs to be added into manifest only when a new *.po file was downloaded. So all of these languages would never be added into manifest.
I could solve this by adding all these dropped languages into manifest manually and then when bot would run, it would remove all of those, that do not have 50% translation. Also I could just drop those, that do not have 50% translations and those that do I could manually add. But I thought that we can let bots to do it.

Reply:
Ah, thanks. Please re-title the commit to "Drop all languages that are not currently in the shell manifest". "displayed in cockpit" is really wrong and confusing, as these languages really *do* get displayed in the UI (if the browser language matches), just not in the language selection dialog.